packages/README.md

```diff
@@ -1,3 +1,81 @@
 # Zarf Packages
 
 This folder contains packages maintained by the [Zarf team](https://github.com/defenseunicorns/zarf/graphs/contributors).  Some of these packages are used by `zarf init` for new cluster initialization.
+
+**Packages**
+- [distros](#distros)
+- [gitea](#gitea)
+- [logging-pgl](#logging-pgl)
+- [zarf-agent](#zarf-agent)
+- [zarf-registry](#zarf-registry)
+
+### Distros
+
+The distros package adds optional capabilities for spinning up and tearing down clusters.  Currently, the following distros are supported:
+
+- [EKS](https://aws.amazon.com/eks/) - Zarf deploys and tears down using the `eksctl` binary under the hood. See how it's done in the EKS package's [`zarf.yaml`](./distros/eks/zarf.yaml) and checkout the [EKS package's config](./distros/eks/eks.yaml) for more information.
+
+- [k3s](https://k3s.io/) - Zarf deploys and tears down using the `k3s` service under the hood. See how it's done in the k3s package's [`zarf.yaml`](./distros/k3s/common/zarf.yaml).
+
+
+#### Usage Examples  
+  
+**EKS**  - Create/Deploy EKS cluster. (requires `eksctl` credentials)
+
+```bash
+zarf package create packages/distros/eks -o build --confirm
+
+zarf package deploy build/zarf-package-distro-eks-multi-x.x.x.tar.zst --components=deploy-eks-cluster --set=CLUSTER_NAME='zarf-nightly-eks-e2e-test',INSTANCE_TYPE='t3.medium' --confirm
+```
+
+See the [nightly-eks test](../.github/workflows/nightly-eks.yml) for another example.
+
+**k3s** - Create/Deploy a k3s cluster.
+
+```bash
+# Linux with `systemd` and `root` access only (no `sudo`)
+zarf init --components=git-server
+```
+
+### Gitea
+
+Users who rely heavily on GitOps find it useful to deploy an internal Git repository.  Zarf uses [Gitea](https://gitea.io/en-us/) to provide this functionality.  The Gitea package deploys a Gitea instance to the cluster and configures it to use the credentials in the `private-git-server` secret in the zarf namespace.
+
+_usage_
+
+```bash
+zarf init --components=git-server
+```
+
+### Logging PGL
+
+The Logging PGL package deploys the Promtail Grafana & Loki stack which aggreates logs from different containers and presents them in a web dashboard.  This is useful as a quick way to get logging into a cluster when you otherwise wouldn't be bringing over a logging stack.
```

Comment:
```suggestion
The Logging PGL package deploys the Promtail, Grafana, and Loki stack which aggregates logs from different containers and presents them in a web dashboard.  This is useful as a quick way to get logging into a cluster when you otherwise wouldn't be bringing over a logging stack.
```

Reply:
TY